openshift-maven-plugin/doc/src/main/asciidoc/inc/_introduction.adoc

```diff
@@ -144,7 +144,7 @@ mvn oc:resource oc:deploy
 
 By default a _Service_ and a _Deployment_ object pointing to the created Docker image is created. When running in OpenShift mode, a _Service_ and _DeploymentConfig_ which refers the _ImageStream_ created with `oc:build` will be installed.
 
-Of course you can bind all those jkube.goals to execution phases as well, so that they are called along with standard lifecycle goals like `install`. For example, to bind the building of the Kubernetes resource files and the Docker images, add the following goals to the execution of the f-m-p:
+Of course you can bind all those jkube.goals to execution phases as well, so that they are called along with standard lifecycle goals like `install`. For example, to bind the building of the Kubernetes resource files and the Docker images, add the following goals to the execution of the kmp:
```

Comment:
We're in OpenShift documentation, see line 5.
Same applies here, `oc-m-p` `o-m-p` or any other permutation should be replaced by `omp` (for consistency with the `kmp` chosen in Kubernetes Maven Plugin docs).